Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocCellAttributeRenderingContext.cs

```diff
@@ -0,0 +1,42 @@
+﻿// // This file is part of the re-motion Core Framework (www.re-motion.org)
+// // Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// //
+// // The re-motion Core Framework is free software; you can redistribute it
+// // and/or modify it under the terms of the GNU Lesser General Public License
+// // as published by the Free Software Foundation; either version 2.1 of the
+// // License, or (at your option) any later version.
+// //
+// // re-motion is distributed in the hope that it will be useful,
+// // but WITHOUT ANY WARRANTY; without even the implied warranty of
+// // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// // GNU Lesser General Public License for more details.
+// //
+// // You should have received a copy of the GNU Lesser General Public License
+// // along with re-motion; if not, see http://www.gnu.org/licenses.
+// //
+namespace Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation.Rendering
+{
+  /// <summary>
+  /// Provides forwarding of additional attributes to  <see cref="BocColumnRenderingContext"/>
+  /// </summary>
+  public struct BocCellAttributeRenderingContext<TBocColumnDefinition>
+      where TBocColumnDefinition : BocColumnDefinition
+  {
+    private readonly BocColumnRenderingContext<TBocColumnDefinition> _renderingContext;
+
+    public TBocColumnDefinition ColumnDefinition
+    {
+      get { return _renderingContext.ColumnDefinition; }
+    }
+
+    public BocCellAttributeRenderingContext (BocColumnRenderingContext<TBocColumnDefinition> renderingContext)
+    {
+      _renderingContext = renderingContext;
+    }
+
+    public void AddAttributeToRender (string name, string value)
+    {
```

Comment:
please add:
argument checks.
overload with bool fEncode
overload with PlainTextString for value.
overload with HtmlTextWriterAttribute for name

new method:
AddStyleAttributeToRender(string name, string? value)
AddStyleAttributeToRender(HtmlTextWriterStyle name, string? value)

We basically mirror the Attribute/Style API on HtmlTextWriter.